docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -1,170 +1,85 @@
+.. doctest-skip-all
```

Comment:
Would it be a pain to add `hvpy` to the doc and/or test dependencies and run these doctests?

Reply:
Done and updated the file to pass.